### DISCUSSION TITLE
math formula issues

### DISCUSSION THREAD

**Discussion by iamliukaifang** [SeRazon/Mathplus#8] [Q&A] [answered]
Thanks for your extension Mathplus first, I pretty like it. Here, I found some issues maybe helpful to improve Mathplus in the future:

1. Can't get braces:
 $\{, \}, \$, \& $ <---- `$\{, \}, \$ $,\ \& $`.
 $\{x\} $<----`$\{x\}$` 
    

1. Strange problem. if I put the following equation into one line, they fail to show correctly

    $\sum_{i=1}^{n}$,   $\prod_{i=1}^{n}$ <----`$\sum_{i=1}^{n}$,   $\prod_{i=1}^{n}$`,
    
    But If I separate them into two lines with an empty line, they look good:
    
    $\sum_{i=1}^{n}$,   
    
    $\prod_{i=1}^{n}$
    
    (text is here)
    ```
    $\sum_{i=1}^{n}$,   
    
    $\prod_{i=1}^{n}$,
    ```

**Comment by SeRazon** [comment, marked as answer]
Problem known, pending resolution. 
It will probably be fixed by 8 months.
